test-clients/src/main/resource/contract/solidity/AliasedOperator.sol

```diff
@@ -0,0 +1,18 @@
+pragma solidity ^0.8.12;
+
+contract AliasedOperator {
+    address public token;
+
+    function initialize(address _token) public {
+        token = _token;
+    }
+
+    function transfer(address to, uint value) external {
+        (bool success, bytes memory data) = token.call(abi.encodeWithSelector(bytes4(keccak256(bytes('transfer(address,uint256)'))), to, value));
+        require(success && (data.length == 0 || abi.decode(data, (bool))), 'UniswapV2: TRANSFER_FAILED');
```

Comment:
Why would you add the "UniswapV2" message? It has nothing to do with the E2E test